subprojects/docs/src/docs/release/notes-template.md

```diff
@@ -2,11 +2,11 @@ The Gradle team is excited to announce Gradle @version@.
 
 This release features [1](), [2](), ... [n](), and more.
 
-We would like to thank the following community members for their contributions to this release of Gradle:
 <!-- 
 Include only their name, impactful features should be called out separately below.
  [Some person](https://github.com/some-person)
 -->
+We would like to thank the following community members for their contributions to this release of Gradle:
```

Comment:
Do you also need to update the release notes?

Reply:
It's fine to not have this comment in release notes, right? Looks like someone else deleted it when he wrote the release notes.

Reply:
So the release notes need to be updated on `master` when this has been merged.